routersploit/modules/exploits/netgear/dgn2200_ping_cgi_rce.py

```diff
@@ -0,0 +1,85 @@
+import requests
+
+from routersploit import (
+    exploits,
+    print_error,
+    print_success,
+    print_status,
+    mute,
+    validators,
+)
+
+
+class Exploit(exploits.Exploit):
+    """
+    Exploits Netgear DGN2200 RCE vulnerability in ping.cgi
+    """
+    __info__ = {
+        'name': 'Netgear DGN2200 RCE',
+        'description': 'Exploits Netgear DGN2200 RCE vulnerability in the ping.cgi script',
+        'authors': [
+            'SivertPL',  # vulnerability discovery
+            'Josh Abraham',  # routesploit module
```

Comment:
It would be cool to add contact information like e-mail address (example: <marcin.bury[at]reverse-shell.com>) or website address